src/template.js

```diff
@@ -30,18 +32,18 @@ switch (data.method) {
       break;
     }
 
-    if (queryPermission('inject_script', data.searchInsightsSource)) {
+    if (queryPermission('inject_script', INSIGHTS_LIBRARY_URL)) {
       injectScript(
-        data.searchInsightsSource,
+        INSIGHTS_LIBRARY_URL,
         data.gtmOnSuccess,
         data.gtmOnFailure,
-        data.searchInsightsSource
+        INSIGHTS_LIBRARY_URL
       );
     } else {
       logger(
         'The library endpoint is not allowed in the "Injects Scripts" permissions.\n\n' +
           'You need to add the value: "' +
-          data.searchInsightsSource +
+          'https://cdn.jsdelivr.net/npm/search-insights*' +
           '"\n\n' +
           'See https://www.simoahava.com/analytics/custom-templates-guide-for-google-tag-manager/#step-4-modify-permissions'
       );
```

Comment:
This actually won't likely happen unless when user imports Algolia template, and deliberately removes the permission section while importing.